### DISCUSSION TITLE
Welcome to job-space-rebuild Discussions!

### DISCUSSION THREAD

**Discussion by kenzo-ruki** [kenzo-ruki/job-space-rebuild#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kenzo-ruki** [comment]
I have to solve the block issue.

**Comment by mindstar0209** [comment]
what does it mean `block`?

**Comment by kenzo-ruki** [comment]
Have you check my project?

**Comment by mindstar0209** [comment]
yes, you mean red and green space?

**Comment by kenzo-ruki** [comment]
right, this page has to be auto scrolling.

**Comment by mindstar0209** [comment, marked as answer]
I just fixed it
you can see my screen

**Comment by kenzo-ruki** [comment]
awesome, cool
